hack/run-functional-test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#Copyright 2021 The hostpath provisioner Authors.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+KUBEVIRT_PROVIDER=k8s-1.25
```

Comment:
I thought we get this passed as env variable?

Reply:
Apparently not, I checked, I think I need to do something else in the Makefile to have this show up in the script. But that is for a future PR. I don't like having the export in several scripts.